### PR TITLE
Rob dev

### PR DESCRIPTION
**Pull request by RobertStoelhorst**
Hey bro my first PR, 

I had a little bit of fun with this and It just came out of nowhere man. I have no idea how I did it !

I have set up a way to dynamically fetch data while in development. simulating an api request, Happy to talk through it with you.

I will admit that this is probably over-engineered but it was something I wanted to handle in a smart way in respect of rate limiting for calls to the API, Just another method on top of caching in the backend as you mentioned. I always feel guilty every time I save a change in my code editor while I am developing and endlessly sending HTTP requests. Talk soon 

I have also added moment js package for UTC date-times and also wrapped the client with an error boundary fallback UI

### CODE REVIEW

**Comment by eustin** [comment]
Hey @RobertStoelhorst! 

So sorry for the late reply. Been so tired with interviews. Some days I've been sleeping for 12 hours!!!

This is great work man. Thanks heaps for doing it. Just one little change involving removing a comment. 

You're awesome 💪

**Comment by RobertStoelhorst** [comment]
Hey bro, No worries at all man, I hope you will get a good outcome soon. 

I have had a mental week and been putting in too many hours in front of the screen I am starting to go cross-eyed. Good things are happening tho.

So I am not familiar with this UI did you make a change or comment on the code or something from this PR ? just trying to spot the Diff

File: client/src/App.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import { useEffect, useState } from "react";
-import { httpGetLatestLaunches } from "./api/houstonBackend";
+import HoustonApis from "./api/houstonBackend";
+// import moment from "moment";
```
Comment (eustin) [inline client/src/App.js:3]: Hey Rob, please remove this commented code if you're not using it.

```suggestion
```

**Review by eustin — CHANGES_REQUESTED** [review]
Hey Rob, 

I stuffed up and forgot to submit the one change I asked for. Sorry!

**Comment by eustin** [comment]
Hey man, that's good to hear! 

All that's required is to do this on your local `rob-dev` branch:

1. Go to `client/src/App.js` and remove the comment on line 3. 
2. Commit the change
3. Git push, which will update this PR

image


> Hey bro, No worries at all man, I hope you will get a good outcome soon.
> 
> I have had a mental week and been putting in too many hours in front of the screen I am starting to go cross-eyed. Good things are happening tho.
> 
> So I am not familiar with this UI did you make a change or comment on the code or something from this PR ? just trying to spot the Diff

G

**Comment by RobertStoelhorst** [comment]
Thanks, soo much bro for the guidance, I really love your detailed response. you are really speaking my language. 
What a legend. 

I will merge now, then work in a new branch on the moment.js features for countdown timer etc. 

I am still smashing out a new portfolio website and getting that up to a good state. 
I don't really have a good indication of when I will have another good block of time working on Houston.  

Let's catch up today tho, if you are free?

**Pull request merged**